### PR TITLE
Changed process width in Query.Obstacles

### PR DESCRIPTION
**Pull request by MaciejDengusiak**
### Issues addressed by this PR

Fixes #

### CODE REVIEW

File: Mollier/SAM.Core.Mollier.UI/Modify/AddMollierProcesses.cs [hunk 1]
```diff
@@ -258,6 +304,10 @@ private static Series createProcessPointsSeries(this Chart chart, MollierPoint m
                 series.Tag = "SecondPoint";
             }
             series.MarkerStyle = MarkerStyle.Circle;
+            if(System.Math.Round(mollierPoint.DryBulbTemperature, 1) == 11.3 && System.Math.Round(mollierPoint.RelativeHumidity, 4) == 92.1575)
```
Comment (ZiolkowskiJakub) [inline Mollier/SAM.Core.Mollier.UI/Modify/AddMollierProcesses.cs:307]: To be removed

File: Mollier/SAM.Core.Mollier.UI/Query/Obstacles.cs [hunk 2]
```diff
@@ -87,6 +92,24 @@ private static List<IClosed2D> obstacles_Zone(UIMollierZone uIMollierZone, Chart
 
             return result;
         }
-      
+
+        private static void checkQuantity(Chart chart, ref bool addProcesses, ref bool addPoints)
```
Comment (ZiolkowskiJakub) [inline Mollier/SAM.Core.Mollier.UI/Query/Obstacles.cs:96]: To discuss with Maciek how it works

**Review by michaldengusiak — CHANGES_REQUESTED** [review]
1. tested to add epsilon on Mollier and works fine but when adding epsilon on psychrometric does not work
2. SHR same issue
3. Tools Disable Labels - does not work

File: Mollier/SAM.Core.Mollier.UI/Modify/AddMollierProcesses.cs (hunk 1 above)
Comment (MaciejDengusiak) [reply Mollier/SAM.Core.Mollier.UI/Modify/AddMollierProcesses.cs:307]: Done

**Pull request merged**